### PR TITLE
chore(build): update chromium-bidi to 0.6.1

### PR DESCRIPTION
**Pull request by Lightning00Blade**
Closes https://github.com/puppeteer/puppeteer/pull/12745 
As it includes the change from it as well.

### CODE REVIEW

**Review by OrKoN — APPROVED** [review]

**Comment by Lightning00Blade** [comment]
Its seems that Firefox is not respecting neither the WebDriver Classic or the WebDriver BiDi  `unhandledPromptBehavior`.
Classic way:
```
2024-07-12T13:23:11.960Z puppeteer:webDriverBiDi:SEND ► [
  '{"id":1,"method":"session.new","params":{"capabilities":{"alwaysMatch":{"acceptInsecureCerts":false,"unhandledPromptBehavior":"ignore","webSocketUrl":true}}}}'
]
2024-07-12T13:23:12.845Z puppeteer:webDriverBiDi:RECV ◀ [
  '{"type":"success","id":1,"result":{"sessionId":"b3f64299-5759-4902-a15e-19bafe933ce5","capabilities":{"acceptInsecureCerts":false,"browserName":"firefox","browserVersion":"130.0a1","platformName":"linux","unhandledPromptBehavior":"dismiss and notify","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:130.0) Gecko/20100101 Firefox/130.0","moz:buildID":"20240711215213","moz:headless":true,"moz:platformVersion":"6.6.15-2rodete2-amd64","moz:processID":1867689,"moz:profile":"/tmp/puppeteer_dev_firefox_profile-XXXXXXmfZUzr","moz:shutdownTimeout":60000,"proxy":{}}}}'
]
```
BiDi way:
```
2024-07-12T13:26:04.921Z puppeteer:webDriverBiDi:SEND ► [
  '{"id":1,"method":"session.new","params":{"capabilities":{"alwaysMatch":{"acceptInsecureCerts":false,"unhandledPromptBehavior":{"default":"ignore"},"webSocketUrl":true}}}}'
]
2024-07-12T13:26:05.826Z puppeteer:webDriverBiDi:RECV ◀ [
  '{"type":"success","id":1,"result":{"sessionId":"0461138e-35e5-4a03-9605-27a05a67c1f7","capabilities":{"acceptInsecureCerts":false,"browserName":"firefox","browserVersion":"130.0a1","platformName":"linux","unhandledPromptBehavior":"dismiss and notify","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:130.0) Gecko/20100101 Firefox/130.0","moz:buildID":"20240711215213","moz:headless":true,"moz:platformVersion":"6.6.15-2rodete2-amd64","moz:processID":1869294,"moz:profile":"/tmp/puppeteer_dev_firefox_profile-XXXXXX1egd7n","moz:shutdownTimeout":60000,"proxy":{}}}}'
]
```

**Comment by OrKoN** [comment]
@Lightning00Blade can we skip it for Firefox for now?

**Comment by whimboo** [comment]
I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1907935 to get this fixed. 

It would be good to give us a little bit of time (if possible) to actually investigate such issues when major changes like that are done. 

@orkon, could we at least wait with the next release of Puppeteer until we got it fixed? It's actually a simple issue.

**Comment by OrKoN** [comment]
@whimboo we need to update the dependencies for other work. Whenever the fix is available we will update again.

**Comment by OrKoN** [comment]
@whimboo would it be possible for you to backmerge the fix to 129?

**Comment by OrKoN** [comment]
@Lightning00Blade I actually meant to skip setting the capability for Firefox not the test.

File: test/TestExpectations.json [hunk 1]
```diff
@@ -299,6 +299,13 @@
     "expectations": ["SKIP"],
     "comment": "TODO: add a comment explaining why this expectation is required (include links to issues)"
   },
+  {
+    "testIdPattern": "[dialog.spec] Page.Events.Dialog *",
+    "platforms": ["darwin", "linux", "win32"],
+    "parameters": ["firefox", "webDriverBiDi"],
+    "expectations": ["SKIP"],
```
Comment (whimboo) [inline test/TestExpectations.json:306]: Was there actually a reason why these were marked as `SKIP` and not `FAIL`? I'll provide a PR in a moment which will remove this entry again.

Comment (OrKoN) [reply test/TestExpectations.json:306]: Thanks for flagging. @Lightning00Blade could you please check if we can work around by not setting the capability for Firefox yet and roll back the test expectation changes?

Comment (whimboo) [reply test/TestExpectations.json:306]: You can leave it as is. I've a fix which landed and will be in the next Nightly build that is going to be available in the next 2-3h.

**Comment by whimboo** [comment]
> @whimboo would it be possible for you to backmerge the fix to 129?

Good point. I'm going to request the uplift to beta.

**Comment by whimboo** [comment]
FYI https://github.com/puppeteer/puppeteer/pull/12755 is reverting all the test expectations.

**Comment by OrKoN** [comment]
@whimboo please let us know if the fix can be uplifted to beta. Otherwise, we would need a workaround on our side.

**Comment by whimboo** [comment]
> @whimboo please let us know if the fix can be uplifted to beta. Otherwise, we would need a workaround on our side.

I don't see a reason why not. We are at beta 5 so a couple of beta's still around and this is a low risk patch.

**Pull request merged**